docs-src/docs/kognic-io/scenes/cameras.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Cameras
+---
+A `Cameras`  consists of a single frame of camera images, where the frame can contain between 1-9 images from different sensors. For more documentation on what each field corresponds to in the `Cameras` object please check the section related to [Input Overview](/docs/kognic-io/overview).
```

Comment:
Should the section be called Scene Overview?

Reply:
Yes